docker-compose.yml

```diff
@@ -24,5 +24,6 @@ services:
     restart: on-failure
     environment:
```

Comment:
guild_id env is missing

Reply:
da wir aber sowieso für multi guild entwickeln, ist die frage ob wir das überhaupt drin lassen wollen

Reply:
Also für Development brauchen wir es definitiv, sonst wird man ja irgendwann wahnsinnig. Ich glaube ich würde es aber auch aus Customization-Gründen drinnen lassen bzw. mit reinnehmen

Reply:
stimmt